server/repository/src/diesel_macros.rs

```diff
@@ -95,6 +95,60 @@ macro_rules! apply_simple_string_filter {
     }};
 }
 
+/// Example expand, when called with:
+///
+/// ```
+/// apply_simple_string_or_filter!(query, filter.code_or_name, item_dsl::code, item_dsl::name)
+/// ```
+///
+/// ```
+/// if let Some(string_filter) = filter.code_or_name {
+///     if let Some(value) = equal_filter.equal_to {
+///         query = query.filter(item_dsl::code.eq(value));
+///         query = query.or_filter(item_dsl::name.eq(value));
+///     }
+///
+///     if let Some(value) = equal_filter.like {
+///         query = query.filter(item_dsl::code.like(format!("%{}%", value)));
+///         query = query.or_filter(item_dsl::name.like(format!("%{}%", value)));
+///     }
+/// }
+/// ```
+#[cfg(not(feature = "postgres"))]
+macro_rules! apply_simple_string_or_filter {
```

Comment:
bit of an offroad - adding in the ability to filter on two fields. have implemented for items (another offroad) and will wire that up in another PR for the item list